docs/comparisons/README.md

```diff
@@ -0,0 +1,9 @@
+# Comparisons between .NET Standard and other platforms
+
+Many have asked for a diff between .NET Standard and other existing platforms.
```

Comment:
This is not always the case. In Core, Stream.CopyTo has been made virtual, many parameter names are different, etc.